src/components/VenueCard.js

```diff
@@ -22,17 +33,35 @@ export const VenueCard = ({ venueName, state, city, url, venuePhone, bookingEmai
           className='emailer'
           >Send Email to {venueName}</a>
         </div>
-      )
-    } else if(info && title !== 'Booking Email'){
+      );
+    } else if (classTitle === 'website'){
+      return  info !== '' && isUrlValid(info) === true
+        ?
+        <a
+          target="_blank"
+          href={url}
+          className={classTitle}
+          rel="noopener noreferrer"
+        >{url}</a>
+        :
+        <a
+          target='_blank'
```

Comment:
A new window is opened with a google search and the venue name, city and state already searched if there is no website already provided